### PR TITLE
LPD-3113 Trying to create a new user via API with a longer than allowed firstName produces a nonfriendly error message in the logs

### PR DESCRIPTION
**Pull request by michelevigilante**
https://liferay.atlassian.net/browse/LPD-3113

### CODE REVIEW

File: portal-impl/src/com/liferay/portal/service/impl/UserLocalServiceImpl.java [hunk 1]
```diff
@@ -7176,6 +7176,9 @@ protected void validateFullName(
 		if (Validator.isNull(firstName)) {
 			throw new ContactNameException.MustHaveFirstName();
 		}
+		else if (firstName.length() > 75) {
+			throw new ContactNameException.MustHaveValidFirstName();
```
Comment (smottal) [inline portal-impl/src/com/liferay/portal/service/impl/UserLocalServiceImpl.java:7180]: I don't like this name :) . Does something like `.Invalid***` already exist?

**Comment by smottal** [comment]
@michelevigilante there are two things needed:
1) an exception mapper
2) an integration test that checks the new scenario

**Comment by michelevigilante** [comment]
ci:test:sf

**Comment by michelevigilante** [comment]
ci:test:relevant

**Comment by michelevigilante** [comment]
ci:test:relevant

**Comment by michelevigilante** [comment]
ci:test:sf

**Comment by michelevigilante** [comment]
ci:test:relevant

**Review by smottal — APPROVED** [review]

**Comment by smottal** [comment]
ci:forward

**Comment by michelevigilante** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:report:17691001

**Comment by michelevigilante** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:report:20126101

**Comment by smottal** [comment]
ci:forward:force

**Pull request closed without merging**